File: SDKs/neutrinoapi.net/3.5.0/typescript/src/sdk/models/shared/phoneplaybackresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class PhonePlaybackResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=calling" })
  calling: boolean;

  @SpeakeasyMetadata({ data: "json, name=numberValid" })
  numberValid: boolean;
}

File: SDKs/neutrinoapi.net/3.5.0/python/src/sdk/models/shared/phoneplaybackresponse.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class PhonePlaybackResponse:
    calling: bool = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('calling') }})
    number_valid: bool = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('numberValid') }})
    
